### PR TITLE
added auto assign and PR labeling Github actions

### PR DESCRIPTION
**Pull request by camoore1979**

### CODE REVIEW

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# A list of reviewers, split into different groups, to be added to pull requests (GitHub user name)
+reviewGroups:
+  groupA:
+    - BWortman
+    - camoore1979
+    - korymarks
+  groupB:
+    - babennettdev
+    - jeffbum
+    - khgriffi259
```
Comment (jeffbum) [inline .github/auto_assign.yml:23]: not adding tyler here?

File: .github/workflows/assign-and-label.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      - uses: kentaro-m/auto-assign-action@v1.1.0
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+  pr-labeler:
+    name: "Auto label the Pull Reqest"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: TimonVS/pr-labeler-action@v3
+        with:
+          configuration-path: .github/pr-labeler.yml # optional, .github/pr-labeler.yml is the default value
```
Comment (jeffbum) [inline .github/workflows/assign-and-label.yml]: if optional && it's already the default value, do we need this line?

**Review by jeffbum — APPROVED** [review]
Couple questions in here.

File: .github/auto_assign.yml (hunk 1 above)
Comment (camoore1979) [reply .github/auto_assign.yml:23]: smh

File: .github/workflows/assign-and-label.yml (hunk 2 above)
Comment (camoore1979) [reply .github/workflows/assign-and-label.yml]: removed

File: .github/auto_assign.yml (hunk 1 above)
Comment (camoore1979) [reply .github/auto_assign.yml:23]: fixed

**Pull request merged**